www-client/firefox/firefox-111.0.ebuild

```diff
@@ -790,7 +790,7 @@ src_configure() {
 	# For future keywording: This is currently (97.0) only supported on:
 	# amd64, arm, arm64 & x86.
 	# Might want to flip the logic around if Firefox is to support more arches.
-	if use ppc64; then
+	if use ppc64 || use riscv; then
 		mozconfig_add_options_ac '' --disable-sandbox
```

Comment:
I'd like to see separated commit for '--disable-sandbox', and also moving keywording riscv commit to last of this series

Reply:
Okay